Magento_Theme/web/js/custom-script.js

```diff
@@ -0,0 +1,18 @@
+let noRouteParagraph = document.querySelector('p');
+
+let noRouteText = noRouteParagraph.innerHTML;
+
```

Comment:
don't add empty lines between variables declaration. it's a bit hard for reading